cmd/livepeer/starter/starter.go

```diff
@@ -1225,6 +1225,22 @@ func StartLivepeer(ctx context.Context, cfg LivepeerConfig) {
 					if *cfg.Network != "offchain" {
 						n.SetBasePriceForCap("default", core.Capability_AudioToText, config.ModelID, autoPrice)
 					}
+					n.SetBasePriceForCap("default", core.Capability_AudioToText, config.ModelID, autoPrice)
+
+				case "llm-generate":
```

Comment:
@kyriediculous can simply become `llm` with new changes in https://github.com/livepeer/ai-worker/pull/137/files.